README.rst

```diff
@@ -0,0 +1,99 @@
+Before the start, make sure that you have an account on `PyPI`_
+
+
+Quickstart
+----------
+ 
+1. Install Cookiecutter
+
+Create and activate an environment for the package project.
+
+.. code-block:: bash
+
+    conda create -n <repo_name> python=<python_version>
+
+Activate your environment.
+
+.. code-block:: bash
+
+    activate <repo_name>          #Windows
+    source activate <repo_name>   #Linux
+    
+Install cookiecutter.
+
+.. code-block:: bash
+
+    conda install cookiecutter
+
+
+2. Create the package
+
+Use cookiecutter, informing the address of `cookiecutter-esss-pypackage`
+
+.. code-block:: bash
+
+    cookiecutter cookiecutter https://github.com/ESSS/cookiecutter-esss-pypackage
+
+You'll be asked to enter:
```

Comment:
Simply state: "Answer the questions in the prompt" without listing what will be asked (no point in listing it, and can get out of sync quickly)

Reply:
Done